CMakeLists.txt

```diff
@@ -150,6 +152,23 @@ if(ROBOTOLOGY_ENABLE_TELEOPERATION)
   find_or_build_package(walking-teleoperation)
 endif()
 
+# Human Dynamics Estimation
+if(ROBOTOLOGY_ENABLE_HUMAN_DYNAMICS)
+  if (WIN32)
+        set(ROBOTOLOGY_ENABLE_ICUB_HEAD TRUE)
+        set(ROBOTOLOGY_USES_CFW2CAN TRUE)
+        set(ROBOTOLOGY_USES_ESDCAN TRUE)
+        find_or_build_package(forcetorque-yarp-devices)
+  endif()
+  find_or_build_package(wearables)
+  if(NOT WIN32)
+    find_or_build_package(osqp)
+    find_or_build_package(OsqpEigen)
+    find_or_build_package(human-dynamics-estimation)
```

Comment:
All of this do not compile on Windows?

Reply:
`osqp` and `osqpEigen` are added here, since only it is necessary of `human-dynamics-estimation`. Otherwise, there is no dependency on that. 
I should add it here, `master` branch currently does not depend on these packages, these dependencies are in `devel` branch which soon will be merged to `master`.

Reply:
If these are dependencies of `human-dynamics-estimation` they can just be added as dependencies in the `Buildhuman-dynamics-estimation.cmake` file, and then they will be automatically compiled or not depending of whether `human-dynamics-estimation` is compiled or not. 
Do not worry if they are not needed now, compiling something more is not a big problem.

Reply:
Sorry @kouroshD , why this was marked as resolved? There is any specific reason why human-dynamics-estimation is not added on Windows?

Reply:
I saw you delete the Windows option for building it. 
I cannot recall now, but probably because there were some problem to run it on Windows.
I have not recently run `human-dynamics-estimation` in Windows, maybe @Yeshasvitvs  or @lrapetti  can provide more info.
 In any case, we do not have visualization in Windows if I am not wrong, and normally we need to visualize.

Reply:
Yes, indeed there were some problems for compilation on Windows, and  now they are solved thanks to @lrapetti , see https://github.com/robotology/human-dynamics-estimation/issues/160 .